config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
-  - manager.yaml
+- manager.yaml
```

Comment:
These changes shouldn't be committed. AFAIK they get changes during the local build, deployment.